src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.html

```diff
@@ -60,50 +60,113 @@
-                        : ('artemisApp.assessment.feedbackCommentPlaceholder' | artemisTranslate)
-                "
-                id="feedback-editor-text-input"
-                [(ngModel)]="feedback.detailText"
-                #detailText
-                (keyup)="textareaAutogrow()"
-                (keydown.escape)="escKeyup()"
-                (focus)="inFocus()"
-                [readOnly]="readOnly && !isConflictingFeedbackEditable"
-                (ngModelChange)="didChange()"
-            ></textarea>
-            <div class="card-header" *ngIf="!conflictMode && feedback.credits !== 0 && textBlock?.numberOfAffectedSubmissions && feedback.type === FeedbackType.MANUAL">
-                <fa-icon [icon]="faInfoCircle" size="1x" class="text-info" placement="left"></fa-icon>
-                <span
-                    jhiTranslate="artemisApp.textAssessment.feedbackEditor.impactWarning"
-                    [ngbTooltip]="'artemisApp.textAssessment.feedbackEditor.impactWarningTooltip' | artemisTranslate"
-                    placement="bottom"
-                    class="impactWarningLabel"
-                    (mouseenter)="mouseEnteredWarningLabel()"
-                    [translateValues]="{ affectedSubmissionsCount: textBlock.numberOfAffectedSubmissions }"
-                    >This feedback might affect <b>{{ textBlock.numberOfAffectedSubmissions }}</b> other similar submission/s.</span
-                >
+        <div class="input-group">
+            <div class="input-group-prepend">
+                <div *ngIf="criteria" class="dropdown" ngbDropdown>
+                    <button class="btn btn-primary dropdown-toggle" ngbDropdownToggle type="button"></button>
+                    <!--if there is only one criterion, dropdown-menu contains its instructions-->
+                    <ul *ngIf="criteria.length == 1" class="dropdown-submenu-single" id="test" ngbDropdownMenu>
+                        <li *ngFor="let instruction of criteria[0]!.structuredGradingInstructions">
+                            <a (click)="updateAssessmentWithDropdown(instruction)" class="dropdown-item" [style.background-color]="setInstrColour(instruction)">
+                                <table class="table layout-table">
+                                    <tbody>
+                                        <tr>
+                                            <td class="td-score fw-bold">{{ instruction!.credits }}P</td>
+                                            <td>
+                                                <span class="td-instruction">{{ instruction!.instructionDescription }}</span>
+                                            </td>
+                                            <td class="td-usage-count">
+                                                <span *ngIf="instruction!.usageCount && instruction!.usageCount !== 0; else usageCountInfinity">{{ instruction!.usageCount }}</span>
```

Comment:
0 is already falsy.  (same below)
```suggestion
                                                <span *ngIf="instruction!.usageCount; else usageCountInfinity">{{ instruction!.usageCount }}</span>
```